scratch/sim_correlator.py

```diff
@@ -81,6 +84,18 @@ def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
     parser.add_argument(
         "--narrowband-beams", type=int, default=0, help="Number of dual-polarisation narrowband beams [%(default)s]"
     )
+    parser.add_argument("--vlbi", action="store_true", help="Enable VLBI mode [no]")
+    parser.add_argument(
+        "--vlbi-recv-pols",
+        type=comma_split(str, 2),
+        metavar="[+-]P,[+-]P",
+        required=True,
+        help="Input polarisations for VLBI (±x, ±y, ±L or ±R)",
+    )
+    parser.add_argument(
+        "--vlbi-send-station-id", type=str, default="me", help="VDIF Station ID for VLBI output [%(default)s]"
```

Comment:
I didn't know how helpfuly you wanted this `help` statement to be. I was going to add something like "Two-character VDIF station ID [...]". What do you reckon?

Reply:
I wouldn't worry. If someone is overriding the default, then they know what a station ID is.